### ISSUE TITLE
kubeconfig file on jumpbox is broken

### ISSUE THREAD

**Issue by mfabiani-av** [Azure/aks-engine#1552]
**Describe the bug**
The kubeconfig on the jumpbox is broken: it contains "\n" wherever there should be a new line, and all the ' " ' characters are escaped:

```{\n        \"apiVersion\": \"v1\",\n        \"clusters\": [\n            {\n                \"cluster\": {\n                    \"certificate-authority-data\": ...```

**Steps To Reproduce**
Create a cluster with this configuration:
``` "privateCluster":  
        {  
          "enabled": true,  
          "jumpboxProfile":  
          {  
            "name": "internalappsjump",  
            "vmSize": "Standard_D2_v3",  
            "publicKey": "ssh-rsa ..........",  
            "username": "xxxxxxx"  
          }  
        },  
```
NOTE: I'm running aks-engine in Windows Linux Subsystem (Ubuntu)

**Expected behavior**
```{
     "apiVersion": "v1",
     "clusters": [
                        {"cluster": {
                              "certificate-authority-data": ...
```

**AKS Engine version**
```$ aks-engine version
Version: v0.36.5
GitCommit: 6b8255b54
GitTreeState: dirty
```

**Kubernetes version**
1.12.8

**Additional context**

**Comment by CecileRobertMichon** [comment]
@mfabiani-av thanks for reporting, sounds like a recent regression. I've labeled this as good first issue since it should be an easy one to repro + fix. If anyone wants to pick this up and needs guidance feel free to @ me in a comment.

**Comment by Alwandy** [comment]
@CecileRobertMichon I'll pick it up since I need to use jump box myself too.. I'll fork and work my way through or any tips?

**Comment by CecileRobertMichon** [comment]
@Alwandy thanks! I would start by trying to repro to confirm the bug. Then, you will probably want to start by looking at https://github.com/Azure/aks-engine/blob/master/parts/k8s/kubeconfig.json and https://github.com/Azure/aks-engine/blob/master/parts/k8s/cloud-init/jumpboxcustomdata.yml#L16.

**Comment by Alwandy** [comment]
@mfabiani-av 
I ran the latest version of AKS with private cluster and could successfully create my private jumpbox and connect to it.

![image](https://user-images.githubusercontent.com/5923107/60669247-c07fd780-9e65-11e9-8e42-a111b1cb72f0.png)

@CecileRobertMichon Seems to be no problem as it creates the jump box fine and configures it fine.

**Comment by Alwandy** [comment]
I successfully reproduced the issue. 

Will investigate on a fix now.

**Comment by Alwandy** [comment]
Looks to be related to `masterVars["kubeconfig"] = escapeSingleLine(kubeConfig)` since it's already being escaped during generating, there's no need to escape it again. 

Replacing with `masterVars["kubeconfig"] = kubeConfig` and trying again to see if it resolves the issue.

**Comment by mfabiani-av** [comment]
Hello,
has this fix made it to the latest versions?  I'm running Version: v0.38.4 but still have the same problem.

**Comment by CecileRobertMichon** [comment]
@mfabiani-av not yet, https://github.com/Azure/aks-engine/commit/86030eb128ac6528674f7ca69f5d083f0b99f677 isn't included in v0.38. It will be included in the upcoming v0.39.0.
